commerce/domain/inventory/cart/calculator/classes/InventoryCartCalculatorSampleTest.cls

```diff
@@ -0,0 +1,150 @@
+
+        /**
+         * @description This constructor should only be exposed to customers in a test context
+         */
+        public DefaultInventoryCartCalculatoMockExecutor() {}
+
+        public override void defaultInventory(CartExtension.CartCalculateCalculatorRequest request) {
+            // avoid the call to the implementation
+            return;              
+        }
+        
+    }
+    
+    private static CartExtension.Cart arrangeAndLoadCartWithSpecifiedStatusAndThreeItems(CartExtension.CartStatusEnum cartStatus, boolean ignoreRedShirtSku) {
+        Id cartId = arrangeCartWithSpecifiedStatus(cartStatus);
+        arrangeThreeCartItems(cartId, ignoreRedShirtSku);
+        return CartExtension.CartTestUtil.getCart(cartId);
+    }
+    
+    private static ID arrangeCartWithSpecifiedStatus(CartExtension.CartStatusEnum cartStatus) {
+        Account account = new Account(Name = ACCOUNT_NAME);
+        insert account;
+
+        WebStore webStore = new WebStore(Name = WEBSTORE_NAME, OptionsCartCalculateEnabled = true);
+        insert webStore;
+
+        WebCart webCart = new WebCart(
+                Name = CART_NAME,
+                WebStoreId = webStore.Id,
+                AccountId = account.Id,
+                Status = cartStatus.name());
+        insert webCart;
+        return webCart.Id;
+    }
+
+    private static List<ID> arrangeThreeCartItems(ID cartId, Boolean ignoreRedShirtSku) {
+        CartDeliveryGroup deliveryGroup = new CartDeliveryGroup(Name = DELIVERYGROUP_NAME, CartId = cartId);
+        insert deliveryGroup;
+
+        CartItem cartItem1 = new CartItem(
+                Name = CART_ITEM1_NAME,
+                CartId = cartId,
+                CartDeliveryGroupId = deliveryGroup.Id,
+                Quantity = 3,
+                SKU = SKU1_NAME,
+                Type = CartExtension.SalesItemTypeEnum.PRODUCT.name());
+        insert cartItem1;
+
+        CartItem cartItem2 = new CartItem(
+                Name = CART_ITEM2_NAME,
+                CartId = cartId,
+                CartDeliveryGroupId = deliveryGroup.Id,
+                Quantity = 3,
+                SKU = SKU2_NAME,
+                Type = CartExtension.SalesItemTypeEnum.PRODUCT.name());
+        insert cartItem2;
+
+        CartItem cartItem3 = new CartItem(
+            Name = CART_ITEM3_NAME,
+            CartId = cartId,
+            CartDeliveryGroupId = deliveryGroup.Id,
+            Quantity = 3,
+            SKU = SKU_RED_SHIRT,
+            Type = CartExtension.SalesItemTypeEnum.PRODUCT.name());
+
+        if (ignoreRedShirtSku) {
+            insert cartItem3;
+        } 
+
+        if (ignoreRedShirtSku) {
+            return new List<ID>{cartItem1.Id, cartItem2.Id};
+        } else {
+            return new List<ID>{cartItem1.Id, cartItem2.Id, cartItem3.Id};
+        }
+        
+    }
+
+    private static void assertNoCartValidationOutputs(CartExtension.Cart cart, Integer expectedCVOCount) {
```

Comment:
I think you'd want to change the name of this method to `assertCartValidationOutputs`. Remove "No" because you provide expected count as a parameter.